### ISSUE TITLE
`send` needs upgraded

### ISSUE THREAD

**Issue by shatran** [zeit/next.js#8208]
# Bug report

## Describe the bug

When running `ts-node -r tsconfig-paths/register src/server` for a local development of a ts project, the following error occur:
```
Error: Attempt to change mapping for "wasm" extension from "application/wasm" to "application/wasm". Pass `force=true` to allow this, otherwise remove "wasm" from the list of extensions for "application/wasm".
    at Mime.define (.../node_modules/mime/Mime.js:54:15)
    at Object.<anonymous> (.../node_modules/next-server/dist/server/serve-static.js:8:21)
```

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. run ts-node with tsconfig-paths (to support tsconfig paths attributes, ts-node does not support it by default)

## Expected behavior
Paths support to a ts project should be allowed. I would expect to be able to run ts-node together with tsconfig-paths

## System information

- OS: macOS
- Version of Next.js: 9.0.3-canary.1

## Additional context

Iv'e raised a PR to solve this issue: https://github.com/zeit/next.js/pull/8209

**Comment by Timer** [comment]
Hey! This is likely due to some sort of double-evaluation, however, we cannot know that for sure because a reproducible demo was not provided.

Regarding this specific error, it sounds like upgrading `send` to its latest version is the best approach, then removing our line that defines a `wasm` type.

Can you please send a PR doing this?

**Comment by Timer** [comment]
Fixed in 9.0.4.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
